blocks/init/src/blocks/components/drawer/assets/index.js

```diff
@@ -0,0 +1,11 @@
+document.addEventListener('DOMContentLoaded', () => {
```

Comment:
Aren't we using that `domReady` method?
Or this is something similar in function to that method

Reply:
yes like @gabriel-glo  use this

https://developer.wordpress.org/block-editor/packages/packages-dom-ready/